institution based logo

randyhoulahan opened the pull request.

sbilodeau reviewed (changes requested):
You dropped template padding added during COP
https://github.com/scbd/cctv/pull/17/files#diff-507ef303e241a4f81f4e0227234f3e9f05015713360bd2cade13b0ab7f2b4876L43

randyhoulahan commented:
> 

It was a conflict when I added my stash.  I thought I included it.   Not sure what  went wrong.  

> You dropped template padding added during COP https://github.com/scbd/cctv/pull/17/files#diff-507ef303e241a4f81f4e0227234f3e9f05015713360bd2cade13b0ab7f2b4876L43



> You dropped template padding added during COP https://github.com/scbd/cctv/pull/17/files#diff-507ef303e241a4f81f4e0227234f3e9f05015713360bd2cade13b0ab7f2b4876L43

Thanks, an error on conflict resolution.   Corrected.
